content/docs/editing.md

```diff
@@ -185,6 +185,12 @@ The cloned document is the same document as its original one, but with the separ
 
 You can find more on cloning to other views or instances in the [Views > Move / Clone](../views/#move-clone) section, and more on manipulating tabs through the tab bar in [Other Resources > Tabs](../other-resources/#tabs).
 
+## Virtual Space
+
+When editing a text document in Notepad++, there is potentially a blank area beyond the end of each line, which has no characters (not even space characters) in the document, because it's passed the end-of-line characters.  By default, if you tried to click in that empty space, the caret (typing cursor) would move to the end of the line, instead of being placed where you clicked.  However, if you enabled **[Settings > Preferences > Editing 1](../preferences/#editing-1) > `☐ Enable virtual space`**, then if you clicked in that empty space, Notepad++ would place the caret there, beyond the end of the line; if you then typed some text, Notepad++ would fill the empty space between the end of the line and the caret with spaces, and the text you typed would go after those spaces.  (The Virtual Space feature was added in Notepad++ v8.4.3, so if you have an older Notepad++, you would need to upgrade to use this feature.)
```

Comment:
I would do "which has no characters (not even space characters), because it's past the end-of-line characters."

Reply:
In general, seems like a great job to me!